client/me/developer/main.jsx

```diff
@@ -1,6 +1,7 @@
 import { recordTracksEvent } from '@automattic/calypso-analytics';
 import { ToggleControl } from '@wordpress/components';
 import classnames from 'classnames';
+import cookie from 'cookie';
```

Comment:
Rather than using a cookie, would `savePreference` not be more appropriate here? That would apply more per account than browser.

Reply:
We need to remove `dismissed` status after 1 day, so it would be more complicated approach for such a thing.